src/m365/spo/commands/theme/theme-set.ts

```diff
@@ -98,8 +98,8 @@ class SpoThemeSetCommand extends SpoCommand {
       if (contents && contents.ErrorInfo) {
         throw contents.ErrorInfo.ErrorMessage || 'ClientSvc unknown error';
       }
-      return Promise.resolve();
-    } 
+      return;
```

Comment:
hmmm, TBH I don't think we need this at all. We may just remove `return` if it's void